bin/install.js

```diff
@@ -30,6 +30,13 @@ if ('next' in projectDependencies) {
     );
   }
 }
+if (
```

Comment:
let's add:

- an extra line here
- a comment above the `if` to describe why we need this (mention Netlify)

Reply:
Added a line and a comment in this commit
[6e09664](https://github.com/upleveled/eslint-config-upleveled/pull/228/commits/6e09664d93ed24b94952e9a214c0315d03dcea37)
Also edited the version number in the comment
[c8f207f](https://github.com/upleveled/eslint-config-upleveled/pull/228/commits/c8f207fb7e43198b102a9fd5ab719cc4694d30a6)